packages/client/src/i18n/messages/views/userSetup.ts

```diff
@@ -360,6 +366,41 @@ const messagesToDefine: IUserSetupMessages = {
     defaultMessage: 'Username Reminder Requested',
     description: 'Description for User username requested audit action',
     id: 'user.profile.auditList.usernameRequested'
+  },
+  sentForReviewAuditAction: {
+    defaultMessage: 'Sent Incomplete Registration',
+    description: 'Description for sending registration for review audit action',
+    id: 'user.profile.auditList.sentForReview'
+  },
+  reInstatedInProgressAuditAction: {
+    defaultMessage: 'Reinstated to In Progress',
+    description:
+      'Description for sending registration from Reinstated to In progress audit action',
+    id: 'user.profile.auditList.reInstatedToInProgress'
+  },
+  reInstatedInReviewAuditAction: {
+    defaultMessage: 'Reinstated to In Review',
+    description:
+      'Description for sending registration from Reinstated to In review audit action',
+    id: 'user.profile.auditList.reInstatedToInReview'
+  },
+  retrieveAuditAction: {
+    defaultMessage: 'Retrieved',
+    description:
+      'Description for Retrieving registration to In review audit action',
+    id: 'user.profile.auditList.retrieve'
+  },
+  reInStatedRejectedAuditAction: {
+    defaultMessage: 'Reinstated to Require Updates',
+    description:
+      'Description for sending registration from Reinstated to require updates audit action',
+    id: 'user.profile.auditList.reInstatedToInReview'
+  },
+  sentForApprovalAuditAction: {
+    defaultMessage: 'Sent For Approval',
+    description:
+      'Description for sending registration for approval audit action',
+    id: 'user.profile.auditList.sentForApproval'
```

Comment:
@favour234 which PR in Farajaland contains this content and French translation?

Reply:
@euanmillar ..i think @KaliGreen  made most update on this pull request.

Reply:
> @favour234 which PR in Farajaland contains this content and French translation?

see PR :- https://github.com/opencrvs/opencrvs-farajaland/pull/418